src/common/helpers/Utils.ts

```diff
@@ -304,4 +306,92 @@ export default class Utils implements IUtils {
             id
         }
     }
+
+    // References
+    // https://github.com/diafygi/webcrypto-examples/#aes-gcm
+
+    /**
+     * Import passphrase key using AES-CBC 256
+     * @param randomIV
+     * @param passphraseKey
+     */
+    public static async importKey(passphraseKey: string) {
+        const passphrase = new TextEncoder().encode(passphraseKey)
+        const pwHash = await crypto.subtle.digest('SHA-256', passphrase)
+
+        // TODO: IV needs to be random and shared
```

Comment:
@decent-dev I'm moving this to vanilla webcrypto but I'm don't know how to handle the IV parameter, the lib you are using is easier and backend seems to use the same. Please let me know what you think is the way to go.

Reply:
@decent-dev I implemented EVP KDF but to make this work we'll need to use the same lib in Node as well. 

Please review frontend (right now it won't decrypt because Slots API not using same lib)

The lib is here, it has unit tests for Node.
https://github.com/decent-bet/crypto-evpkdf
https://github.com/decent-bet/crypto-evpkdf/blob/master/src/EvpKDF.spec.ts

The WebcryptoUtils in frontend is different, we remove the polyfill for Webcrypto, should be the only diff change between repos.